content/quickstart/Node.js/metrics.md

```diff
@@ -0,0 +1,571 @@
+---
+title: "Metrics"
+date: 2018-08-20T07:12:03-13:00
+draft: false
+class: "shadowed-image lightbox"
+---
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Brief Overview](#brief-overview)
+- [Getting started](#getting-started)
+- [Enable Metrics](#enable-metrics)
+    - [Import Packages](#import-packages)
+    - [Create Metrics](#create-metrics)
+- [Record and Aggregate Data](#record-and-aggregate-data)
+    - [Create Views and Tags](#create-views-and-tags)
+    - [Recording Metrics](#recording-metrics)
+- [Exporting to Stackdriver](#exporting-to-stackdriver)
+
+In this quickstart, we’ll gleam insights from code segments and learn how to:
+
+1. Collect metrics using [OpenCensus Metrics](/core-concepts/metrics) and [Tags](/core-concepts/tags)
+2. Register and enable an exporter for a [backend](/core-concepts/exporters/#supported-backends) of our choice
+3. View the metrics on the backend of our choice
+
+## Requirements
+- [Node.js](https://nodejs.org/) 6 or above and `npm` (already comes with Node.js)
+- Google Cloud Platform account and project
+- Google Stackdriver Monitoring enabled on your project
+
+{{% notice tip %}}
+For assistance setting up Stackdriver, [Click here](/codelabs/stackdriver) for a guided codelab.
+{{% /notice %}}
+
+## Installation
+
+OpenCensus: `npm install @opencensus/nodejs`
```

Comment:
`@opencensus/nodejs` isn't used in this tutorial

OpenCensus: `npm install @opencensus/core`

Reply:
`@opencensus/nodejs` has `@opencensus/core` as a dependency, so when you download oc/nodejs you'll be able to import `@opencensus/core`. The ideia here is to recommend the download of oc/nodejs. But thinking about this now, I think it gets confusing to the user to download something and use other thing. Thanks for pointing that out, I'm going to change.